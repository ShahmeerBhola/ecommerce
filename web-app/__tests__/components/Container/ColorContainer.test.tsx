import React from 'react';
import renderer from 'react-test-renderer';
import ColorContainer from '../../../components/Container/ColorContainer';

describe('ColorContainer component', () => {
  test('it renders properly', () => {
    const tree = renderer
      .create(
        <ColorContainer color="gray-300">
          <p>Hello</p>
        </ColorContainer>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it renders properly with non-default props', () => {
    const tree = renderer
      .create(
        <ColorContainer color="gray-300" className="hello-there" minHeight="500">
          <p>Hello</p>
        </ColorContainer>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
