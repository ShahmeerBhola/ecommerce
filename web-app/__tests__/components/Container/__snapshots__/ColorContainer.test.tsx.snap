// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorContainer component it renders properly 1`] = `
<div
  className="bg-gray-300"
>
  <div
    className="container mx-auto px-5 md:py-20 py-5 min-h-80"
  >
    <p>
      Hello
    </p>
  </div>
</div>
`;

exports[`ColorContainer component it renders properly with non-default props 1`] = `
<div
  className="bg-gray-300"
>
  <div
    className="container mx-auto px-5 md:py-20 py-5 500 hello-there"
  >
    <p>
      Hello
    </p>
  </div>
</div>
`;
